cmd/list.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/codegangsta/cli"
+	"github.com/exercism/cli/api"
+	"github.com/exercism/cli/config"
+)
+
+const msgExplainFetch = "In order to fetch a specific assignment, call the fetch command with a specific assignment.\n\nexercism fetch ruby/matrix"
```

Comment:
Good catch. It actually worked for me to say `exercism fetch ruby/matrix`, but we should be telling people the right thing to do, not the thing that happens to work in this release.